packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -1734,6 +1739,39 @@ class ThemeData with Diagnosticable {
   Color get backgroundColor => _backgroundColor!;
   final Color? _backgroundColor;
 
+  /// Obsolete property that was the default color of the [BottomAppBar].
```

Comment:
Why does this getter for `bottomAppBarColor` change?

Reply:
It's unchanged, moved from its current incorrect location